department-of-veterans-affairs/va.gov-cms
# VACMS-7995: Disabled entity embed module

- Disabled entity embed dependencies in Rich Text and Rich Text limited editors.
- Disabled Entity Embed module.
- Uninstalled Entity Embed module.
- ~Removed the "Entity Embed" module from code base via composer.~  This needs to be done in a separate PR that follows this one.

## Description

closes #7995 

## Testing done
Passes all tests

## Screenshots

**_Entity Embed_ module not listed in "Extend"**
![VACMS-7995-extend-no-entity-embed-module](https://user-images.githubusercontent.com/846871/153949799-560a37af-1141-42f5-a605-da9d2b91aa9b.png)

**No embed buttons in the "Embed buttons" admin page**
![VACMS-7995-embed-buttons-local](https://user-images.githubusercontent.com/846871/153949946-a9d0839b-7db7-4bf5-82cd-22387792b8f1.png)


## QA steps

As user administrator
1. Go to `/admin/modules`:
    - [ ] Search for "embed" to verify and confirm that the "Entity Embed" module is not listed
2. Go to `/admin/config/content/embed`:
    - [ ] Check that there are no embed buttons listed

See above screen shots for reference.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
